### ISSUE TITLE
Sub 1s readings

### ISSUE THREAD

**Issue by charlespax** [PaxInstruments/t400-firmware#169]
It would be nice to take readings at half seconds. Right now we can't hit that mark without extreme optimization. Sending to the ice box.

**Comment by cibomahto** [comment]
It's possible to read and display the data this fast, but writing to SD could possibly take much longer (the write time isn't guaranteed), so this isn't reasonable to do without a fairly major rewrite.

**Comment by charlespax** [comment]
I have several MicroSD card sample from several vendors I'm considering. When recording at 1s intervals is the SD write speed a concern or is pretty much everything capable of working at 1s?

**Comment by cibomahto** [comment]
Good question, I don't know the answer. My guess is that it's the speed of the arduino that's the limiting factor.

**Comment by protological** [comment]
I did some measurements and it looks like we can read 13-14 ADC values per second and the display writing can be done 10 times per second.  I removed unused code and tested each function running as fast as possible in the main loop() function.  If we were to write the SD card once a second with 2 set of data I think we can get to 1/2 second sampling.  This will take a lot of updates but it is possible.
